bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioConsoleCommandExtension.java

```diff
@@ -46,9 +46,9 @@ public AudioConsoleCommandExtension() {
     @Override
     public List<String> getUsages() {
         return Arrays.asList(new String[] {
-                buildCommandUsage(SUBCMD_PLAY + " <sink> <filename> <volume>",
+                buildCommandUsage(SUBCMD_PLAY + " <sink> [<filename> [<volume>]]",
```

Comment:
Reading your code, I understand the case with only one parameter is not the sink but the filename.
So you should put two entries in this list.

Reply:
I see.